cmd/client/service.go

```diff
@@ -1,31 +1,34 @@
 package main
 
 import (
+	"flag"
 	"fmt"
-	"time"
+	"log"
 
-	sq "github.com/Masterminds/squirrel"
-	dbUtil "github.com/monstercat/golib/db"
+	"github.com/monstercat/gologx/logxhost"
 )
 
-type service struct {
-	machine   string
-	name      string
-	last_seen time.Time
-}
-
 func showServices(name string, args []string) error {
+	var postgres string
```

Comment:
We should by default let the program dictate what the postgres Url is through env var, cmd line, or config file.